src/Core/Models/Request/PasswordlessCreateLoginRequest.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using System;
+using Bit.Core.Enums;
+
```

Comment:
⛏️ Remove this given that it's not being used as far as I see

Reply:
the import is being used on the constructor and property for the `AuthRequestType? type`